mkosi/__init__.py

```diff
@@ -891,7 +891,7 @@ def determine_partition_table(args: CommandLineArguments) -> Tuple[str, bool]:
             pn += 1
             run_sfdisk = True
 
-    if not is_generated_root(args):
+    if not is_generated_root(args) and not args.usr_only:
```

Comment:
is_generated_root() returns true for args.usr_only already, this change should not alter anything hence?

Reply:
Oops!